pandas/tests/tseries/test_offsets.py

```diff
@@ -1952,6 +1952,10 @@ def _check_roundtrip(obj):
         _check_roundtrip(self._object(2))
         _check_roundtrip(self._object() * 2)
 
+    def test_copy(self):
```

Comment:
add the issue number (or pr number in this case) as a comment

Reply:
Done.